assets/src/edit-story/components/library/panes/media/common/innerElement.js

```diff
@@ -154,11 +154,10 @@ function InnerElement({
   const imageProps = {
     ...commonImageProps,
     src: thumbnailURL,
-    'aria-label': alt,
+    alt,
```

Comment:
`alt` is already included here as part of `commonImageProps` -> `commonProps`, so this is redundant here.

Reply:
I have now removed the redundant alt here.